### PR TITLE
Prepare for 0216 release

### PR DESCRIPTION
**Pull request by elenavillamil**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [2.0.120230216]
+
+* CLI version update that address [issue](https://github.com/Azure/Bridge-To-Kubernetes/issues/162), [issue 89](https://github.com/Azure/Bridge-To-Kubernetes/issues/89) and [issue 56](https://github.com/Azure/Bridge-To-Kubernetes/issues/56)
```
Comment (Tatsinnit) [inline CHANGELOG.md:5]: 👍 Nice, looks good, please feel free to add information about:

* Added `redact` funct for logging information under [Fix/readact creds from logging](https://github.com/Azure/vscode-bridge-to-kubernetes/commit/73bc3d8218c47eacadadc23d59a5f442346ef28e)

**Review by Tatsinnit — APPROVED** [review]
❤️☕️🙏 Thank you for addition as a review, looks good to me, beside minor addition if you feel like to make it part of changelog.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230124",
+    "version": "2.0.120230216",
```
Comment (hsubramanianaks) [inline package.json:5]: Hi Elena seems like testing team is having issues with this release, let's hold on for more investigation and proceed once it is clear. Thank you.

**Review by hsubramanianaks — CHANGES_REQUESTED** [review]
Hi Elena seems like testing team is having issues with this release, let's hold on for more investigation and proceed once it is clear. Thank you.

File: CHANGELOG.md (hunk 1 above)
Comment (elenavillamil) [reply CHANGELOG.md:5]: Didnt noticed this was already checked in, thanks!

**Comment by elenavillamil** [comment]
> Hi Elena seems like testing team is having issues with this release, let's hold on for more investigation and proceed once it is clear. Thank you.

Yes Joyer mixed thread, he added issue for testing of pull164 into thread for testing this release :) I think we are good on this one.

**Review by hsubramanianaks — APPROVED** [review]

**Pull request merged**